semgrep/semgrep/rule_lang.py

```diff
@@ -131,6 +134,31 @@ def from_string(cls, s: str, filename: Optional[str] = None) -> "Span":
         end = Position(line=len(lines), col=len(lines[-1]))
         return Span(start=start, end=end, file=filename, source_hash=src_hash)
 
+    @classmethod
+    def from_string_token(
+        cls,
+        s: str,
+        line: int,
+        col: int,
+        path: List[Dict[str, Union[int, str]]],
+        filename: Optional[str] = None,
+    ) -> "Span":
+        src_hash = SourceTracker.add_source(s)
+        start = Position(line + 1, col + 1)  # 1-index instead of 0
+        lines = s.splitlines()
+        row_diff = len(lines)
+        col_diff = len(lines[-1])
+        end = Position(line=(row_diff + line - 1), col=(col_diff + col + 1))
+        return Span(
+            start=start,
+            end=end,
+            file=filename,
+            source_hash=src_hash,
+            config_path=tuple(path),
+            config_start=Position(0, 1),
```

Comment:
@brendongo Unclear if this should instead be Position(1, 1)? Or Position(0, 0)?

Reply:
Did you get an answer to this?